NEWS.md

```diff
@@ -89,6 +89,7 @@
 - `spout` classes no longer need to implement `Iterator`, instead they should return `Iterator` of newly introduced `Item` objects from `getItems()` method. The types of properties of items have also been revisited. ([#1341](https://github.com/fossar/selfoss/pull/1341), [#1342](https://github.com/fossar/selfoss/pull/1342))
 
 ### Other changes
+- Amine and others have rewritten the **Android app** from scratch, you will want to install the [new one from F-Droid](https://f-droid.org/packages/bou.amine.apps.readerforselfossv2.android) to keep receiving updates.
```

Comment:
@aminecmi do you think this changelog entry is enough? It does not look like F-Droid has any way to redirect users to the new app, as far as I can glance from the [reference](https://f-droid.org/docs/Build_Metadata_Reference/).

Reply:
This should be enough. 

From what I found, I don't think that there is any way to redirect to the new version.

Even Google would consider the two apps different, as the package name is different.